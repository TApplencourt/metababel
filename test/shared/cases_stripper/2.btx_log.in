event1: { pf_1 = 12 }
event2: { pf_1 = 12 }
