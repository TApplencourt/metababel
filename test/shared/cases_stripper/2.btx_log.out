Trace class:
  Stream class `sc1` (ID 0):
    Supports packets: No
    Supports discarded events: No
    Supports discarded packets: No
    Event class `event1` (ID 0):
      Payload field class: Structure (1 member):
        pf_1: Unsigned integer (64-bit, Base 10)
  Stream class `sc2` (ID 1):
    Supports packets: No
    Supports discarded events: No
    Supports discarded packets: No
    Event class `event2` (ID 0):
      Payload field class: Structure (1 member):
        pf_1: Unsigned integer (64-bit, Base 10)

{Trace 0, Stream class ID 0, Stream ID 0}
Stream beginning:
  Class name: sc1
  Trace:
    Stream (ID 0, Class ID 0)
    Stream (ID 0, Class ID 1)

{Trace 0, Stream class ID 0, Stream ID 0}
Event `event1` (Class ID 0):
  Payload:
    pf_1: 12

{Trace 0, Stream class ID 0, Stream ID 0}
Stream end
